### ISSUE TITLE
CI support (Travis)

### ISSUE THREAD

**Issue by kuraga** [SteveSanderson/knockout-es5#27]
@SteveSanderson Good day! Thank you for this work!

What about Travis?

P.S. See also commits of a fork, they can have some ideas: 
https://github.com/passy/knockout-es5-passy/compare/28a9acc...master

**Comment by kuraga** [comment]
Work now?

**Comment by archangel-irk** [comment]
No, because @SteveSanderson has not yet created an account on travis and did not add the hook for this.
